src/config/config.go

```diff
@@ -126,6 +130,14 @@ func NewConfig(c *cli.Context) (SwanConfig, error) {
 		swanConfig.Janitor.Domain = c.String("domain")
 	}
 
+	if c.String("api-prefix") != "" {
```

Comment:
in my mind the api-perfix is equal to api-version, it is a constant rather than an variable